application/views/admin/authentication/login.php

```diff
@@ -151,6 +154,7 @@
 <!-- Set focus on user input -->
 <script type='text/javascript'>
 $( document ).ready(function() {
+    $('#user').val('<?php echo addslashes($_POST['user'])?>');
```

Comment:
1. Never us $_POST : https://www.yiiframework.com/doc/api/1.1/CHttpRequest#getPost-detail
2. Why in JS ? Add it directly in PHP 
3. this don't fix «when changing language you must send another params, to disable the error.»

Reply:
Removed from JS, added directly in PHP using getPost().
Changing language does not pop up this alert. No need to disable it.
Alert pops up when you press 'Log in' button and name or password is incorrect.

Reply:
> Alert pops up when you press 'Log in' button and name or password is incorrect.

Or is both empty , i just check

Reply:
Yes, if both empty then also pops up (empty is incorrect)
But this is not related with changing language. 
Changing language works without pressing 'Log in' button.

Reply:
Ah right …